austraits_build_tutorial.qmd

```diff
@@ -0,0 +1,625 @@
+# AusTraits.build tutorial
+
+## Introduction
+
+With more than 1.8 million data records, AusTraits is Australia's [largest plant trait database](austraits_database.html#plant_database).
+
+The following tutorial introduces:
+
+-   the database structure, [`traits.build`](https://github.com/traitecoevo/traits.build)
```

Comment:
let's elaborate this slightly, otherwise the link looks like a ink to the relevant part of this tutorial. Maybe something like

```
-   [the database structure](link to section below), created using the [`traits.build`] package(https://github.com/traitecoevo/traits.build)

```

then similar changes for other links